recommend/yaml/rules.yaml

```diff
@@ -1,6 +1,8 @@
+version: v0.1.3
```

Comment:
@vishnusomank Is this `0.1.3` represents something?

Reply:
it is v0.1.0 the current policy-template release

Reply:
As discussed privately, let's make this `0.0.1` since currently these are just sample policies.